src/PKSim.Core/Model/IndividualProteinTask.cs

```diff
@@ -0,0 +1,273 @@
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Plasma), organ.Name, moleculeName, groupName,
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_PLASMA)
+         );
+
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Endosome), organ.Name, moleculeName, groupName,
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_ENDOSOME)
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Intracellular), organ.Name, moleculeName, groupName,
+            relExpParam(REL_EXP),
+            fractionParam(FRACTION_EXPRESSED_INTRACELLULAR, CoreConstants.Rate.PARAM_F_EXP_INTRACELLULAR, editable: false),
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTRACELLULAR)
+         );
+
+         AddContainerExpressionNew(organ.Container(CoreConstants.Compartment.Interstitial), organ.Name, moleculeName, groupName,
+            fractionParam(FRACTION_EXPRESSED_INTERSTITIAL, CoreConstants.Rate.ZERO_RATE),
+            initialConcParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTERSTITIAL)
+         );
+      }
+
+      private ParameterValueMetaData relExpParam(string paramName) => new ParameterValueMetaData
+      {
+         ParameterName = paramName,
+         Dimension = DIMENSIONLESS,
+         DefaultValue = 0,
+         GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         BuildingBlockType = PKSimBuildingBlockType.Individual,
+         IsInput =  true,
+      };
+
+      private ParameterRateMetaData fractionParam(string paramName, string rate, string groupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         bool editable = true) =>
+         new ParameterRateMetaData
+         {
+            ParameterName = paramName,
+            Rate = rate,
+            CalculationMethod = CoreConstants.CalculationMethod.EXPRESSION_PARAMETERS,
+            BuildingBlockType = PKSimBuildingBlockType.Individual,
+            CanBeVaried = true,
+            CanBeVariedInPopulation = true,
+            ReadOnly = !editable,
+            Dimension = CoreConstants.Dimension.Fraction,
+            GroupName = groupName,
+            IsInput = true
+         };
+
+      private ParameterRateMetaData initialConcParam(string rate) =>
+         new ParameterRateMetaData
+         {
+            ParameterName = INITIAL_CONCENTRATION,
+            Rate = rate,
+            CalculationMethod = CoreConstants.CalculationMethod.EXPRESSION_PARAMETERS,
+            BuildingBlockType = PKSimBuildingBlockType.Individual,
+            CanBeVaried = true,
+            CanBeVariedInPopulation = true,
+            Dimension = MOLAR_CONCENTRATION,
+            GroupName = CoreConstants.Groups.RELATIVE_EXPRESSION,
+         };
+
+      protected IParameter createFormulaParameterIn(IContainer parameterContainer, ParameterRateMetaData parameterRateMetaData, string moleculeName,
+         string groupName = null)
```

Comment:
should `createFormulaParameterIn `be moved to the _ParameterFactory_?

Reply:
No I don't think so, It uses the factory to create a parameter but does molecule specific stuff such as replacing keywords etc..

Reply:
however some of the logic will be simplified